src/Slider.jsx

```diff
@@ -270,7 +270,7 @@ class Slider extends React.Component {
 
     const points = Object.keys(marks).map(parseFloat);
     if (step !== null) {
-      const closestStep = Math.round(val / step) * step;
+      const closestStep = (Math.round((val - min) / step) * step) + min;
```

Comment:
Ah okay, this ones fine.  

We should submit a PR for the classnames part, potentially taking their suggestion for fixing if they don't want the PR.  We don't want to end up maintaining our own fork of the component.